cmd/arc_summary/test_arc_summary3.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
```

Comment:
Added a monkey-see, monkey-do test that is basically a straight copy of the one for arc_summary.py with the new parameters, and included it in Makefile and the `linux.run`. Don't see a way to automatically trigger the Python 3 test script test_arc_summary3.py included in `cmd/arc_summary`? That would be nice to do it additionally, because it can test individual functions. Would it make sense to move that script somewhere in the test folder as well?

Reply:
> Would it make sense to move that script somewhere in the test folder as well?

Yes, your going to need to and then add a simply wrapper which calls it with `log_must`.